tests/integration/test_task.py

```diff
@@ -1,3 +1,4 @@
+import multiprocessing
```

Comment:
```suggestion
import multiprocessing
```
```suggestion
import multiprocessing as mp
```

Reply:
Haha, that was my original line, but changed with advice from @bouweandela :)
I did agree with his argument though: because we only use the multiprocessing package once, it only reduces readability and doesn't add much to shorten it.

Reply:
yes, if a module is used <2 times it should be kept with its name, unless of course it's a German name that is made up of >300 letters :grin:

Reply:
This is the standard way to import multiprocessing 😅